src/platform/platform_worker.c

```diff
@@ -300,6 +307,65 @@ CxPlatWorkersUninit(
     CxPlatLockUninitialize(&CxPlatWorkerLock);
 }
 
+#define DYNAMIC_POOL_PROCESSING_TIME 1000000 // 1 second
```

Comment:
"TIME" -> "PERIOD" ("TIME" sounds like amount of time spent processing, and is also confusing next to LastPoolProcessTime)